Gemfile.lock

```diff
@@ -139,10 +139,9 @@ GEM
     ruby2_keywords (0.0.5)
     safe_yaml (1.0.5)
     sass (3.4.25)
-    sass-embedded (1.83.4-arm64-darwin)
-      google-protobuf (~> 4.29)
-    sass-embedded (1.83.4-x86_64-linux-gnu)
+    sass-embedded (1.83.4)
       google-protobuf (~> 4.29)
```

Comment:
This change isn't related and thus shouldn't be in this pull request.

Reply:
Aghhhh my dev environment keeps doing this automatically for no reason :(